lib/upload/pdf.js

```diff
@@ -29,8 +34,6 @@ module.exports = {
     };
 
     // Saving function I'm gonna change
-    const oldCLog = console.log;
-    const oldCError = console.error;
     const oldPError = process.stderr.write;
 
     // Read and catch data from the Progress Bar
```

Comment:
refactor the function you assign to `process.stderr.write` to an external function that you can test. 
Test that it matches the data correctly
Test that it calls the eventEmitter with the right arguments